recipes/microhapulator/run_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+wget -O hg38.fasta.gz http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz
+gunzip hg38.fasta.gz
+faidx hg38.fasta chr1:100-150 > /dev/null
+
+pytest -m 'not known_failing' --cov=microhapulator --pyargs microhapulator
```

Comment:
Ah, never mind, you are running the pytests. That's a lot more testing than a lot of our recipes see. We really need a  way to store the test data though. Hg38 will be pretty big. Let's hope the link never goes away.

Reply:
These are the same tests that are run as part of MicroHapulator's CI build on Travis. For the next update I'm considering a less comprehensive smoke test just for bioconda purposes.